airbrakes/utils.py

```diff
@@ -10,23 +10,28 @@
 MOVE_CURSOR_UP = "\033[F"  # Move the cursor one line up
 
 
-def convert_to_nanoseconds(value) -> int:
-    """Converts seconds to nanoseconds, if `value` is in float."""
+def convert_to_nanoseconds(timestamp_str: str) -> int | None:
```

Comment:
maybe we could do a more sophisticated check? e.g. check if the number passed into here is more than 10^9?

Reply:
yeah perhaps. But the better solution is to just clean the csv data in the first place